libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -403,7 +410,7 @@ describe('Help inline component', () => {
       await checkAriaPropertiesAndAccessibility(
         'Show help content',
         helpPanelTestEl.id,
-        null,
+        'false',
```

Comment:
This test is the only one that needed to be adjusted, but I'm fairly certain it _should_ be changed. I would appreciate a second pair of eyes on my assertion: If `aria-controls` is set, shouldn't `aria-expanded` be set to `'false'` (and not `null`, like it was before)?